discovery-provider/src/queries/confirm_indexing_transaction_error.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import requests
+from src.tasks.index_metrics import get_all_other_nodes
+from src.queries.get_skipped_transactions import setIndexingError
+
+logger = logging.getLogger(__name__)
+
+'''
+Confirms that a transaction is causing an error indexing across the discovery nodes
+Gets all other discovery nodes and makes an api call to check the status of a transaction
+given a blocknumber, blockhash, and transactionhash
+'''
+def confirm_indexing_transaction_error(redis, blocknumber, blockhash, transactionhash, message):
+    all_other_nodes = get_all_other_nodes()
```

Comment:
This methods gets all "other" nodes that do not share the same owner wallet - should we change this to remove that restriction?

Reply:
tbh i think it might be more accurate to check all discprovs period, regardless of this quorum requirement - it makes more sense / gives more insight to the indexing state

Reply:
this actually already does what we want it to do - my bad on the comment.